### PR TITLE
feat: Configure ecms navigation and pages for administration site - EXO-66546 - Meeds-io/MIPs#88 

### PR DESCRIPTION
**Pull request by IlhemEssaadi**

### CODE REVIEW

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/ecms-portal-configuration.xml [hunk 1]
```diff
@@ -123,7 +123,39 @@
           </object>
         </object-param>
       </init-params>
+    </component-plugin><component-plugin>
```
Comment (azayati) [inline ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/ecms-portal-configuration.xml]:
```suggestion
    </component-plugin>
    <component-plugin>
```

File: apps/portlet-transferrules/src/main/webapp/WEB-INF/conf/transferRules/portal/portal/administration/navigation.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <node>
+        <name>security</name>
+        <label>#{portal.administration.security}</label>
+        <icon>fas fa-shield-alt</icon>
+        <node>
+          <name>transfert-rules</name>
+          <label>#{portal.administration.transfert-rules}</label>
+          <icon>fas fa-exchange-alt</icon>
+          <page-reference>portal::administration::transferRules</page-reference>
+
```
Comment (azayati) [inline apps/portlet-transferrules/src/main/webapp/WEB-INF/conf/transferRules/portal/portal/administration/navigation.xml]:
```suggestion
```

File: apps/portlet-transferrules/src/main/webapp/WEB-INF/conf/transferRules/portal/portal/administration/pages.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+-->
+<page-set xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_objects_1_8 http://www.exoplatform.org/xml/ns/gatein_objects_1_8"
+          xmlns="http://www.exoplatform.org/xml/ns/gatein_objects_1_8">
+  <page>
+    <name>transferRules</name>
+    <title>Transfer Rules</title>
+    <access-permissions>*:/platform/administrators</access-permissions>
+    <edit-permission>manager:/platform/administrators</edit-permission>
+    <container id="singlePageApplicationContainer" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
```
Comment (azayati) [inline apps/portlet-transferrules/src/main/webapp/WEB-INF/conf/transferRules/portal/portal/administration/pages.xml]:
```suggestion
    <container id="singlePageApplicationContainer" template="system:/groovy/portal/webui/container/UIContainer.gtmpl" cssClass="singlePageApplication">
```

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/portal/administration/pages.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+-->
+<page-set xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://www.exoplatform.org/xml/ns/gatein_objects_1_8 http://www.exoplatform.org/xml/ns/gatein_objects_1_8"
+          xmlns="http://www.exoplatform.org/xml/ns/gatein_objects_1_8">
+  <page>
+    <name>SiteExplorer</name>
+    <title>Site Explorer</title>
+    <access-permissions>*:/platform/web-contributors</access-permissions>
+    <edit-permission>manager:/platform/web-contributors</edit-permission>
+    <container id="singlePageApplicationContainer" template="system:/groovy/portal/webui/container/UIContainer.gtmpl">
```
Comment (azayati) [inline ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/portal/administration/pages.xml]:
```suggestion
    <container id="singlePageApplicationContainer" template="system:/groovy/portal/webui/container/UIContainer.gtmpl" cssClass="singlePageApplication">
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/ecms-portal-configuration.xml [hunk 5]
```diff
@@ -25,7 +25,7 @@
    xsi:schemaLocation="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd http://www.exoplatform.org/xml/ns/kernel_1_2.xsd"
    xmlns="http://www.exoplatform.org/xml/ns/kernel_1_2.xsd">
 
-	<external-component-plugins>
+  <external-component-plugins>
 		<target-component>org.exoplatform.portal.config.UserPortalConfigService</target-component>
```
Comment (azayati) [inline ecm-wcm-extension/src/main/webapp/WEB-INF/conf/wcm-extension/portal/ecms-portal-configuration.xml]: format that

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**